pages/blog/_slug.vue

```diff
@@ -80,7 +80,7 @@
             class="d-flex align-items-center justify-content-center timeToRead"
           >
             <ClockIcon color="#4f4f4f" />
-            <span>{{ blogData?.readingStats?.text }}</span>
```

Comment:
why is `?` removed?

Reply:
readingTime is now in utils. so it will always have text... so no need of optional chaining.